packages/icon-button/index.js

```diff
@@ -0,0 +1,129 @@
+import React, {Component} from 'react';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import withRipple from '@material/react-ripple';
+import {MDCIconButtonToggleFoundation} from '@material/icon-button';
+
+const {strings} = MDCIconButtonToggleFoundation;
+
+export class IconButtonBase extends Component {
```

Comment:
Why do you have IconButtonBase and IconButton? Is this because of the ripple? Why dont you have a nice clean `withRipple` call like Button and Fab have?

Reply:
It is mainly because the ripple needs to be "unbounded". I'm not sure if there is a cleaner syntax than the one I have now. I'll try look for something better.

Reply:
This is what the change looks like. There is a minor change to withRipple.

https://github.com/material-components/material-components-web-react/pull/154/commits/7496e445de33d206fdba012f4fee93b4fb7e72fc

Reply:
Seems like we can rename `IconButtonBase` to `IconButton` now, should we do that to keep things consistent?